File: poo/Java/UberBlack.java
import java.util.ArrayList;
import java.util.Map;

class UberBlack extends Car {
    Map<String, ArrayList<Map<String,Integer>>> typeCarAccepted;
    ArrayList<String> seatsMaterial;

    public UberBlack(String license, Integer passengers, Account driver, Map<String, ArrayList<Map<String,Integer>>> typeCarAccepted, ArrayList<String> seatsMaterial) {
        super(license, passengers, driver);
        this.typeCarAccepted = typeCarAccepted;
        this.seatsMaterial = seatsMaterial;
    }
}

File: poo/Python/uberBlack.py
from car import Car

class UberBlack(Car):
    typeCarAccepted = []
    seatsMaterial = []

    def __init__(self, license, passengers, driver, typeCarAccepted, seatsMaterial) -> None:
        super().__init__(license, passengers, driver)
        self.typeCarAccepted = typeCarAccepted
        self.seatsMaterial = seatsMaterial